src/bauhaus/bauhaus.c

```diff
@@ -1294,8 +1294,11 @@ gboolean dt_bauhaus_combobox_add_introspection(GtkWidget *widget,
   while(item->name && item->value != start) item++;
   for(; item->name; item++)
   {
-    dt_bauhaus_combobox_add_full(widget, Q_(item->description ? item->description : item->name),
-                                 DT_BAUHAUS_COMBOBOX_ALIGN_RIGHT, GUINT_TO_POINTER(item->value), NULL, TRUE);
+    const char *text = item->description ? item->description : item->name;
+    if(*text)
```

Comment:
`item->name` if the condition of the _for_ loop and `item->description` is the condition in the ternary operator and only used if not-null, so I believe they have both just been checked, but please double verify my reasoning and tell me if it's wrong.

Introspection will generate a `""` description, rather than a NULL if no $DESCRIPTION marker is found in the comment.

Reply:
Ah true, I didn't look broadly enough and missed the part where `item->name` is checked. Everything ok here.